.github/workflows/push.yml

```
@@ -248,13 +247,24 @@ jobs:
   docker-build:
     needs:
       - pre-checks
       - prepare
     runs-on: arc-runners
     timeout-minutes: 45
     if: needs.prepare.outputs.BUILD_CHUNKS
+    permissions:
+      actions: read
+      contents: read
+    outputs:
+      json: ${{ steps.matrix-output.outputs.json }}
```

actionlint (1.7.4)

property "matrix-output" is not defined in object type {cache-check: {conclusion: string; outcome: string; outputs: {string => string}}; dockerargs: {conclusion: string; outcome: string; outputs: {string => string}}; dockerbuild: {conclusion: string; outcome: string; outputs: {string => string}}; gather: {conclusion: string; outcome: string; outputs: {string => string}}; restore-generated-files-cache: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)